lib/membrane/rtp/serializer.ex

```diff
@@ -65,6 +92,38 @@ defmodule Membrane.RTP.Serializer do
     payload = RTP.Packet.serialize(packet, align_to: state.alignment)
     buffer = %Buffer{payload: payload, metadata: metadata}
     state = Map.update!(state, :sequence_number, &rem(&1 + 1, @max_seq_num + 1))
+
+    state = %{
+      state
+      | any_buffer_sent?: true,
+        stats_acc: %{state.stats_acc | timestamp: Time.vm_time(), rtp_timestamp: rtp_timestamp}
+    }
+
     {{:ok, buffer: {:output, buffer}}, state}
   end
+
+  @impl true
+  def handle_other(:send_stats, _ctx, state) do
+    {stats, state} = get_updated_stats(state)
+    state = %{state | any_buffer_sent?: false}
+    {{:ok, notify: {:serializer_stats, stats}}, state}
+  end
+
+  @spec get_updated_stats(State.t()) :: {Stats.t(), State.t()}
+  defp get_updated_stats(%State{any_buffer_sent?: false} = state) do
+    {:no_stats, state}
+  end
+
+  defp get_updated_stats(%State{stats_acc: stats} = state) do
+    {stats, state}
+  end
```

Comment:
This actually doesn't update stats, `get_stats` would be more accurate. Also, I don't see any reason to return state if it never changes

Reply:
yes, thats right, done.